src/askui/models/model_router.py

```diff
@@ -122,7 +130,7 @@ def hf_spaces_handler() -> HFSpacesHandler:
         ModelName.ASKUI__COMBO: askui_model_router,
         ModelName.ASKUI__OCR: askui_model_router,
         ModelName.ASKUI__PTA: askui_model_router,
-        ModelName.ANTHROPIC__CLAUDE__3_5__SONNET__20241022: anthropic_facade,
+        ModelName.ANTHROPIC__CLAUDE__SONNET__4__20250514: anthropic_facade,
```

Comment:
```suggestion
        ModelName.ANTHROPIC__CLAUDE__3_5__SONNET__20241022: anthropic_facade,
        ModelName.ANTHROPIC__CLAUDE__SONNET__4__20250514: anthropic_facade,
```

Same here?

I think this would omit a breaking change.